### PR TITLE
New vehicle data WindowStatus

### PR DESCRIPTION
**Pull request by atiwari9**
This proposal is to add new vehicle data `WindowStatus` to know the position of windows in a vehicle.

### CODE REVIEW

**Comment by atiwari9** [comment]
@theresalech  - Ready for review

File: proposals/NNNN-New-vehicle-data-WindowStatus.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+# New vehicle data WindowStatus
+
+* Proposal: [SDL-NNNN](NNNN-New-vehicle-data-WindowStatus.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | Policy Server | SHAID | iOS | Java Suite | RPC]
+
+## Introduction
+
+This proposal is to add new vehicle data `WindowStatus` to know position of windows in a vehicle.
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-WindowStatus.md]:
```suggestion
This proposal is to add new vehicle data `WindowStatus` to know the position of windows in a vehicle.
```

File: proposals/NNNN-New-vehicle-data-WindowStatus.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+## Motivation
+
+In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. Position of window is a useful information for micro weather apps, it tells the app if window is open and at what postion.
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-WindowStatus.md]:
```suggestion
In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. The position of windows is useful information for micro weather apps; it tells the app if a window is open and at what position.
```

File: proposals/NNNN-New-vehicle-data-WindowStatus.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+```
+
+## Potential downsides
+
+Author is not aware of any downsides to proposed solution. This proposal just enhances the SDL content.
+
+## Impact on existing code
+
+* SDL Core needs to be updated as per new API.
+* iOS/Java Suite needs to be updated to support getters/setters as per new API.
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-WindowStatus.md]:
```suggestion
* iOS/Java Suite need to be updated to support getters/setters as per new API.
```

File: proposals/NNNN-New-vehicle-data-WindowStatus.md (hunk 1 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-WindowStatus.md]: Done

File: proposals/NNNN-New-vehicle-data-WindowStatus.md (hunk 2 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-WindowStatus.md]: Done

File: proposals/NNNN-New-vehicle-data-WindowStatus.md (hunk 3 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-WindowStatus.md]: Done

**Comment by atiwari9** [comment]
@theresalech - I have updated the proposal as per your comments.

**Comment by atiwari9** [comment]
@theresalech  - I have pushed another commit to implement grids for this proposal as well. Can you please review this again when you get time.

File: proposals/NNNN-New-vehicle-data-WindowStatus.md [hunk 4]
```diff
@@ -25,31 +25,41 @@ We need to add `WindowStatus` for `GetVehicleData`, `SubscribeVehicleData`, `Uns
...
+	<param name="gatesWindowStatus" type="GateWindowStatus" array="true" minsize="0" maxsize="100" mandatory="false">
 		<description>Status of trunk/liftgate window position</description>
 	</param>		
 </struct>
 ```
+
+#### Add new struct `DoorWindowStatus`:
+```xml	
+<struct name="DoorWindowStatus" since="X.x">
+	<description>Describes the Status of a window of a door.</description>
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-WindowStatus.md]: should "Status" be lowercase?

Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-WindowStatus.md]: Done

**Comment by atiwari9** [comment]
@theresalech - I have updated the proposal. Please review.

**Pull request merged**